Porting/cmpVERSION.pl

```diff
@@ -196,21 +197,24 @@ sub pm_file_from_xs {
         push @{$module_diffs{$_}}, $_;
     } elsif (/\.xs\z/ && !/\bt\b/) {
         push @{$module_diffs{pm_file_from_xs($_)}}, $_;
+    } elsif (!/\bt\b/ && /\.[ch]\z/ && m!^((?:dist|ext|cpan)/[^/]+)/!) {
```

Comment:
Why not include test changes?

Reply:
A typo fix might not be worth a bump, but a larger documentation change might be.

Test changes, at least for `dist/`, might fix a failure when the module is released to CPAN.

If we anything in `cpan/` is customized (tests, installed code or data, documentation), we should probably use a `_01` version in core so upstream can easily distinguish our bugs from theirs.

I do a [similar test in Imager](https://github.com/tonycoz/imager/blob/master/xt/x90cmpversion.t) - it complains for any committed change in the sub-modules.

All that said, this is an improvement.